test/javascripts/acceptance/admin-user-emails-test.js.es6

```diff
@@ -0,0 +1,129 @@
+import { acceptance } from "helpers/qunit-helpers";
+
+acceptance("Admin - User Emails", { loggedIn: true });
+
+const response_with_secondary = secondary_emails => {
+  return [200, { "Content-Type": "application/json" }, {
+    id: 1,
+    username: "eviltrout",
+    email: "eviltrout@example.com",
+    secondary_emails: secondary_emails
+  }];
+};
+
+const assert_no_secondary = assert => {
+  assert.equal(
+    find(".display-row.email .value a").text(),
+    "eviltrout@example.com",
+    "displays primary email"
+  );
+  assert.equal(
+    find(".display-row.secondary-emails .value").text().trim(),
+    "No secondary emails",
+    "displays no secondary emails"
+  );
+};
+
+QUnit.test("my account has no secondary emails", assert => {
+  server.get("/admin/users/1.json", () => { // eslint-disable-line no-undef
+    return response_with_secondary([]);
+  });
+  visit("/admin/users/1/eviltrout");
+
+  andThen(() => {
+    assert_no_secondary(assert);
+  });
+});
+
+QUnit.test("my account has one secondary email", assert => {
+  server.get("/admin/users/1.json", () => { // eslint-disable-line no-undef
+    return response_with_secondary(["eviltrout1@example.com"]);
+  });
+  visit("/admin/users/1/eviltrout");
+
+  andThen(() => {
+    assert.equal(
+      find(".display-row.email .value a").text(),
+      "eviltrout@example.com",
+      "displays primary email"
+    );
+    assert.equal(
+      find(".display-row.secondary-emails .value a").text(),
+      "eviltrout1@example.com",
+      "displays secondary email"
+    );
+  });
+});
+
+const assert_multiple_secondary = assert => {
+  assert.equal(
+    find(".display-row.email .value a").text(),
+    "eviltrout@example.com",
+    "displays primary email"
+  );
+  assert.equal(
+    find(".display-row.secondary-emails .value li:first-of-type a").text(),
+    "eviltrout1@example.com",
+    "displays first secondary email"
+  );
+  assert.equal(
+    find(".display-row.secondary-emails .value li:last-of-type a").text(),
+    "eviltrout2@example.com",
+    "displays second secondary email"
+  );
+};
+
+QUnit.test("my account has multiple secondary emails", assert => {
+  server.get("/admin/users/1.json", () => { // eslint-disable-line no-undef
+    return response_with_secondary([
+      "eviltrout1@example.com",
+      "eviltrout2@example.com"
+    ]);
+  });
+  visit("/admin/users/1/eviltrout");
+
+  andThen(() => {
+    assert_multiple_secondary(assert);
+  });
+});
+
+const no_secondary = (assert, button) => {
+  visit("/admin/users/1234/regular");
+  click(`.display-row.${button} button`);
+
+  andThen(() => {
+    assert_no_secondary(assert);
+  });
+};
+
+QUnit.test("has no secondary emails - clicking primary button", assert => {
+  no_secondary(assert, "email");
+});
+
+QUnit.test("has no secondary emails - clicking secondary button", assert => {
+  no_secondary(assert, "secondary-emails");
+});
+
+const multiple_secondary = (assert, button) => {
```

Comment:
Is this test required since we already have `assert_multiple_secondary` ?

Reply:
Yes, without it we'd only be testing that secondary emails are rendered correctly when an admin is viewing their own profile. We wouldn't be testing to see that secondary emails are rendered correctly when an admin is viewing another profile.